src/atoms/Loader/loader-style.scss

```diff
@@ -0,0 +1,59 @@
+@import '../../styles/variables/variables';
```

Comment:
Please import all resources inside scss files, not only the variables. More things will be included there.